.github/workflows/release-please.yml

```diff
@@ -0,0 +1,21 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  Please:
+    runs-on: ubuntu-latest
+
+    steps:
+      - id: release
+        name: Release
+        uses: google-github-actions/release-please-action@v4
+        with:
+          command: manifest
+          config-file: .github/release-please-config.json
+          manifest-file: .github/release-please-manifest.json
+          release-type: elixir
+          token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
```

Comment:
I _think_ you're going to need a PAT for this to work 🤔

Reply:
@teamon, I have been trying to contact you on Slack and X. What would be a good way to reach out?